MOLH/MOLH.swift

```diff
@@ -153,6 +153,16 @@ open class MOLH {
             }
         }
     }
+    
+    open class func reset(transition:UIViewAnimationOptions) {
```

Comment:
@deya-eldeen thank you , can you modify the original reset function to reuse this one, to avoid duplicates

Reply:
i just wanted to make this portable for people when updating pods, like avoid breaking changes when updating... this is why i used overloading, they will have the original function, and are able to use the one that offers animation.

Reply:
@deya-eldeen you are right , keep the two functions , but am talking about the implementation the old function now can use the new one(the one you added).

Reply:
but the signature is different!
reset(transition:UIViewAnimationOptions)
reset()
so you are basically saying


```
    open class func reset() {
        if let delegate = UIApplication.shared.delegate {
            if delegate is MOLHResetable {
                (delegate as!MOLHResetable).reset()
            }
            UIView.transition(with: ((delegate.window)!)!, duration: 0.5, options: transition, animations: {}) { (f) in
            }
        }
    }
    
    open class func reset(transition:UIViewAnimationOptions) {
        if let delegate = UIApplication.shared.delegate {
            if delegate is MOLHResetable {
                (delegate as!MOLHResetable).reset()
            }
            UIView.transition(with: ((delegate.window)!)!, duration: 0.5, options: transition, animations: {}) { (f) in
            }
        }
    }
```

if so, then we can only keep `open class func reset()` with the new implementation, correct ?

Reply:
the above wont even work, because there no transition in it...

just tell me what to do to make another pull request...

what about we make 

```
    open class func reset(transition:UIViewAnimationOptions) {
        if let delegate = UIApplication.shared.delegate {
            if delegate is MOLHResetable {
                (delegate as!MOLHResetable).reset()
            }
            UIView.transition(with: ((delegate.window)!)!, duration: 0.5, options: transition, animations: {}) { (f) in
            }
        }
    }
```

and that it.... one function.

Reply:
now it's a single function that has the functionality... i pushed another commit.